desktop/hot-path.js

```diff
@@ -0,0 +1,6 @@
+import resolveAssets from './resolve-assets'
+const hot = process.env.HOT === 'true'
+
+export default path => {
+  return (hot ? 'http://localhost:4000/dist/' : resolveAssets('dist') + '/') + path
```

Comment:
Will this be running in prod too?  What if the machine running the Keybase app already has another server listening on port 4000?